src/value.rs

```diff
@@ -35,11 +43,10 @@ pub enum Value {
   F64(f64),
```

Comment:
isize means that compiled in a 64-bit sys it is i64, if compiled in a 32-bit sys it is i32